orc8r/cloud/go/Makefile

```diff
@@ -9,6 +9,12 @@ TOOL_DEPS:= \
 	github.com/vektra/mockery/cmd/mockery \
 	github.com/wadey/gocovmerge \
 	gotest.tools/gotestsum
+
+gen::
+	# obsidian/swagger/gen.go depends on obsidian/swagger/protos/gen.go, so it
+	# must be generated first.
+	go generate ./obsidian/swagger/protos
+
```

Comment:
I don't think this change is necessary, please correct me if I'm wrong!

You can verify with `cd ${MAGMA_ROOT}/orc8r/cloud/docker && ./build.py --generate`

Reply:
For posterity: after discussion Hunter, we moved the swagger.Specs interface swagger/spec.Loader, which also removed the generate dependency on missing pb.go files during a full regen.